backend/courses/management/commands/loadstatus.py

```diff
@@ -19,22 +25,56 @@ def set_all_status(semester=None):
         if section_code is None:
             continue
 
+        course_status = status.get("status")
+        if course_status is None:
+            continue
+
+        course_term = status.get("term")
+        if course_term is None:
+            continue
+        if any(course_term.endswith(s) for s in ["10", "20", "30"]):
+            course_term = translate_semester_inv(course_term)
+
+        # Ignore sections not in db
         try:
             _, section = get_course_and_section(section_code, semester)
         except (Section.DoesNotExist, Course.DoesNotExist):
             continue
-        section.status = status["status"]
-        section.save()
+
+        # Resync database (doesn't need to be atomic)
+        last_status_update = section.last_status_update
+        current_status = section.status
+
+        # Change status attribute of section model (might want to use bulk update)
+        if current_status != course_status:
+            section.status = course_status
+            section.save()
+
+        # Add corresponding status update object
+        if add_status_update and last_status_update.new_status != course_status:
```

Comment:
This does resyncing manually, but this means we would have to run this on a corn all the time right? This definitely solves one problem, but we still probably need to do something to fix the underlying issue (why do they mismatch ever?)